writer.go

```diff
@@ -76,3 +76,33 @@ func open(paths []string) ([]zapcore.WriteSyncer, func(), error) {
 	}
 	return writers, close, errs.AsError()
 }
+
+// OpenWriters combines the passed io.Writer objects that are compatible with
+// zapcore.WriteSyncer into a locked WriteSyncer. It returns any errors
+// encountered.
+func OpenWriters(writers ...io.Writer) (zapcore.WriteSyncer, error) {
```

Comment:
I don't think this function should be here - it somewhat defeats the purpose of static typing. Instead, there is a function in this package that can turn a Writer into a WriteSyncer, and then a user can pass the converted Writer to OpenWriteSyncers